2018-11-23 20:45:29,794 INFO 9009[guizhou_restart.py:18 stop_cmd]: --------------------------------------------------------------------------------------------
2018-11-23 20:45:29,794 INFO 9009[guizhou_restart.py:19 stop_cmd]: now stopping the host : 10.11.5.79
2018-11-23 20:45:29,806 INFO 9009[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-23 20:45:29,858 INFO 9009[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-23 20:45:29,895 INFO 9009[transport.py:1746 _log]: Authentication (password) successful!
2018-11-23 20:45:30,272 INFO 9009[guizhou_restart.py:25 stop_cmd]: cmd is : sudo /data/CI/kds-meta-fusion_34_13280/kds-meta-fusion.sh stop
2018-11-23 20:45:30,273 INFO 9009[guizhou_restart.py:26 stop_cmd]: error list stop is :[]
2018-11-23 20:45:30,273 INFO 9009[guizhou_restart.py:27 stop_cmd]: stdout list stop is :[u'/data/CI/kds-meta-fusion_34_13280\n', u'kds-meta-fusion.jar\n', u'/data/CI/kds-meta-fusion_34_13280/lib/kds-meta-fusion.jar (13280) shutdown.\n']
2018-11-23 20:45:30,273 INFO 9009[guizhou_restart.py:28 stop_cmd]: process is stopping,wait a momunt
Traceback (most recent call last):
  File "/home/op/mesh_deploy_5.80/guizhou_restart.py", line 47, in <module>
    start_cmd(ip, krs_sh)
  File "/home/op/mesh_deploy_5.80/guizhou_restart.py", line 32, in start_cmd
    logger.info("now start the host %s:" %ip)
AttributeError: 'function' object has no attribute 'info'
[Finish] Failed to deploy [462]